samples/05_content_publishers/hey_gis_give_me_a_map_of_the_recent_natural_disasters.ipynb

```diff
@@ -1,5 +1,15 @@
 {
```

Comment:
<p>Can you move the ToC under the title?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/521/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>